### PR TITLE
initial proposed ERD layout

### PR DESCRIPTION
**Pull request by mccordgh**
Worked with the group to setup what we believe to be the correct schema for our database.

## Description
This lays outs the tables for Customer, Payment, Product, and Order

## Related Issue
#10

### CODE REVIEW

**Review by asimonia — COMMENTED** [review]
change zip and phone_number to VARCHARs

**Review by nchemsak — COMMENTED** [review]
change expiration date to date field

**Review by rtwhitfield84 — COMMENTED** [review]
add total price field to order. I love github ❤️

**Review by nchemsak — COMMENTED** [review]
change price type to REAL

**Comment by mccordgh** [comment]
Edits complete
@asimonia 👍 
@nchemsak 👍 
@rtwhitfield84 👍

**Comment by mccordgh** [comment]
Forgot to push updated screenshot with last commit. Added it in commit above.

File: ERD/bangazon_proposed.sql [hunk 1]
```diff
@@ -9,6 +9,7 @@ CREATE TABLE Customers
 	state			VARCHAR(45)		NOT NULL,
 	zip				VARCHAR(45)		NOT NULL,
 	phone_number	VARCHAR(45)			NULL,
+	email			VARCHAR(45)		NOT NULL UNIQUE,
```
Comment (rtwhitfield84) [inline ERD/bangazon_proposed.sql]: constraint needs this syntax
CONSTRAINT name_unique UNIQUE (email)

File: ERD/bangazon_proposed.sql [hunk 2]
```diff
@@ -45,11 +46,10 @@ CREATE TABLE Products
```
Comment (mccordgh) [inline ERD/bangazon_proposed.sql]: product needs quantity field.

**Review by mccordgh — COMMENTED** [review]
product table needs quantity field
🌁

**Pull request merged**